aptos-move/framework/aptos-framework/sources/delegation_pool.move

```diff
@@ -956,19 +990,36 @@ module aptos_framework::delegation_pool {
     public entry fun set_operator(
         owner: &signer,
         new_operator: address
-    ) acquires DelegationPoolOwnership, DelegationPool, GovernanceRecords {
+    ) acquires DelegationPoolOwnership, DelegationPool, GovernanceRecords, Beneficiary {
         let pool_address = get_owned_pool_address(signer::address_of(owner));
         // synchronize delegation and stake pools before any user operation
         // ensure the old operator is paid its uncommitted commission rewards
         synchronize_delegation_pool(pool_address);
         stake::set_operator(&retrieve_stake_pool_owner(borrow_global<DelegationPool>(pool_address)), new_operator);
     }
 
+    /// Allows an operator to change its beneficiary.
+    public entry fun set_beneficiary_for_operator(operator: &signer, new_beneficiary: address) acquires Beneficiary {
```

Comment:
Can you add some documentation here that changing the beneficiary would also mean any existing unpaid commission would be paid to the new beneficiary as well?

Reply:
Done.